lib/commands/build.dart

```diff
@@ -58,9 +58,6 @@ class BuildTpkCommand extends BuildSubCommand with TizenExtension {
 
   /// See: [android.validateBuild] in `build_validation.dart`
   void validateBuild(TizenBuildInfo tizenBuildInfo) {
-    if (tizenBuildInfo.targetArchs.contains('aarch64')) {
```

Comment:
Found this after approving, maybe we can block aarch64 for native projects?

Reply:
The current limitation (#83) not only applies to app builds but also to plugin builds, so `build.dart` is not a good place to add the error condition.

Reply:
I fixed the aarch64 native build. The iot-headed-6.0 rootstrap is required.